src/man/firejail.txt

```diff
@@ -2030,6 +2030,14 @@ For details, see
 #903
 .UE
 
+.TP
+\fB\-\-keep-shell-rc
```

Comment:
The options are sorted alphabetically in this file; move it below `--keep-dev-shm`.

Reply:
Makes sense. I moved it bellow --keep-fd to keep the alphanumerical ordering.